scriptmodules/supplementary/wifi.sh

```diff
@@ -59,7 +59,9 @@ function connect_wifi() {
     local options=()
     i=0
     _set_interface_wifi up 2>/dev/null
+    dialog --infobox "\nScanning for WiFi networks..." 5 40
```

Comment:
This needs to be `dialog --infobox "\nScanning for WiFi networks..." 5 40 >/dev/tty` or it wont work via retropie_setup.sh due to the logging.

Reply:
Should be fine now.